pkg/utils/db.go

```diff
@@ -50,6 +50,20 @@ const (
 	defaultBaseServerID = math.MaxUint32 / 10
 )
 
+// PrintTaskInterval is used to periodically print task status.
+var PrintTaskInterval = 30 * time.Second
+
+func init() {
+	failpoint.Inject("PrintStatusCheckSeconds", func(val failpoint.Value) {
+		if seconds, ok := val.(int); ok {
+			PrintTaskInterval = time.Duration(seconds) * time.Second
+			log.L().Info("set printStatusInterval",
+				zap.String("failpoint", "PrintStatusCheckSeconds"),
+				zap.Int("value", seconds))
+		}
+	})
+}
```

Comment:
I think we can set `PrintStatusCheckSeconds` just in `source_worker.go:Start`. Besides, why do we need to set this variable in `utils.go`?